### ISSUE TITLE
Colour contrast in the cp

### ISSUE THREAD

**Issue by angrybrad** [craftcms/cms#1044]
> Created by: Steven Grant (hello@digitalevangelist.net) on 2015/03/10 17:09:13 +0000
> Votes at time of UserVoice import: 4

-----------------------------------

Working with a disability group today going through their publishing workflow.

Training a disability group this morning in publishing with Craft.

Everyone highly complimentary about the control panel experience (although we didn't look at the new Garnish improvements). 

One of big comments that did come out of a group of 10 editors was the colour contrast on things like field instructions, Matrix block containers and their titles - as it stands, it fails WCAG AA and AAA and some of them were having a hard time seeing the outlines on their laptops.

I know screen contrast etc come into play but they definitely commented on there not being enough colour contrast.

**Comment by brandonkelly** [comment]
Lots of readability improvements were made for Craft 3.
